package.json

```diff
@@ -132,5 +132,9 @@
   },
   "resolutions": {
     "babel-core": "7.0.0-bridge.0"
+  },
+  "jest": {
+    "coverageDirectory": "./coverage/",
+    "collectCoverage": true
```

Comment:
Remove this line. You want to instead invoke `jest --coverage` as your `coverage` script.